frank-flow/src/frontend/src/app/flow/flow.component.ts

```diff
@@ -9,6 +16,65 @@ import { NodeService } from './node/node.service';
 export class FlowComponent {
   nodes = [];
   connections = [];
+  private panZoomConfigOptions: PanZoomConfigOptions = {
+    zoomLevels: 10,
+    zoomStepDuration: 0.2,
+    freeMouseWheelFactor: 0.01,
+    zoomToFitZoomLevelFactor: 0.5,
+    dragMouseButton: 'left',
+    zoomOnDoubleClick: false,
+  };
+  panzoomConfig: PanZoomConfig = new PanZoomConfig(this.panZoomConfigOptions);
 
-  constructor(private nodeService: NodeService) {}
+  @ViewChild('nodeContainer', { read: ElementRef })
+  nodeContainerRef!: ElementRef;
+  private offset = 500;
```

Comment:
Offset is the space added and deleted? Maybe call it:
`private readonly ExpensionSize = 500;`

Reply:
Done